e2e/test/helpers/IotHubConnectionCredentialsExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Text;
+using Microsoft.Azure.Devices.Client;
+
+namespace Microsoft.Azure.Devices.E2ETests.Helpers
+{
+    internal static class IotHubConnectionCredentialsExtensions
+    {
+        public static string GetIotHubConnectionString(this IotHubConnectionCredentials iotHubConnectionCredentials)
```

Comment:
This logic is used by a single edge related test that tests edge capabilities. Moved this logic to a test helper instead of keeping it in our sourcecode.